test/testinfra/MUXTestInfra/Infra/TestEnvironment.cs

```diff
@@ -105,57 +115,77 @@ public static TestApplicationInfo MUXControlsTestApp
         {
             get
             {
-#if USING_TAEF
-                string testAppName = "MUXControlsTestApp_8wekyb3d8bbwe!taef.executionengine.universal.App";
+                // When building inside Visual Studio, we'll install the test app from a directory to avoid needing to build and install an AppX,
+                // since that mimics how F5 deployment works from Visual Studio and is a lot faster.
+                // When building from a command line, we don't have Visual Studio integration, so we'll build and install an AppX in that case.
+#if INSTALL_FROM_APPX
```

Comment:
>INSTALL_FROM_APPX [](http://example.com/codeflow?start=4&length=17)

This makes me cringe a little bit, since it violates the principle of least surprise - I would expect the output of building the dll with msbuild to match building it with VS. I wonder is there another approach we could take?